### PR TITLE
Docs: Migrate Scenario Guide to collection

### PR DESCRIPTION
**Pull request by Akasurde**
##### SUMMARY

Migrate scenario guide from ansible/ansible repo to
this collection.

Signed-off-by: Abhijeet Kasurde <akasurde@redhat.com>

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
docs/docsite/rst/kubernetes_scenarios/k8s_intro.rst
docs/docsite/rst/kubernetes_scenarios/k8s_inventory.rst
docs/docsite/rst/kubernetes_scenarios/k8s_scenarios.rst
docs/docsite/rst/kubernetes_scenarios/scenario_k8s_object.rst
docs/docsite/rst/scenario_guide.rst

### CODE REVIEW

File: docs/docsite/rst/kubernetes_scenarios/scenario_k8s_object.rst [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+Assumptions
+===========
+
+- User has required level of authorization to create, delete and update resources on the given Kubernetes cluster.
+
+Caveats
+=======
+
+- community.kubernetes 1.1.0 is going to migrate to `kubernetes.core <https://github.com/ansible-collections/kubernetes.core>`_
```
Comment (gravesm) [inline docs/docsite/rst/kubernetes_scenarios/scenario_k8s_object.rst:42]:
```suggestion
- community.kubernetes 2.0.0 has been renamed to `kubernetes.core <https://github.com/ansible-collections/kubernetes.core>`_
```

File: docs/docsite/rst/kubernetes_scenarios/scenario_k8s_object.rst [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+.. seealso::
+
+    `Kubernetes Python client <https://github.com/kubernetes-client/python>`_
+        The GitHub Page of Kubernetes Python client
+    `Kubernetes Python client - Issue Tracker <https://github.com/kubernetes-client/python/issues>`_
+        The issue tracker for Kubernetes Python client
+    `OpenShift Python client <https://github.com/openshift/openshift-restclient-python>`_
+        The GitHub Page of OpenShift Dynamic API client
+    `OpenShift Python client - Issue Tracker <https://github.com/openshift/openshift-restclient-python/issues>`_
+        The issue tracker for OpenShift Dynamic API client
```
Comment (gravesm) [inline docs/docsite/rst/kubernetes_scenarios/scenario_k8s_object.rst:84]:
```suggestion
```

**Review by gravesm — CHANGES_REQUESTED** [review]

**Comment by felixfontein** [comment]
You might want to add a call to `antsibull-lint collection-docs` to CI for this. It will run rstcheck on the `.rst` files, make sure that `docs/docsite/extra-docs.yml` looks good, and that the labels in the `.rst` files are correctly prefixed.

**Comment by Akasurde** [comment]
> You might want to add a call to `antsibull-lint collection-docs` to CI for this. It will run rstcheck on the `.rst` files, make sure that `docs/docsite/extra-docs.yml` looks good, and that the labels in the `.rst` files are correctly prefixed.

I will raise a separate CI PR to address this. Thanks for the feedback.

**Review by gravesm — APPROVED** [review]

**Pull request merged**